itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCompactor.java

```diff
@@ -1621,6 +1621,7 @@ public void mmTableOpenWriteId() throws Exception {
     verifyFooBarResult(tblName, 2);
     verifyHasBase(table.getSd(), fs, "base_0000005_v0000016");
     runCleaner(conf);
+    runCleaner(conf);
```

Comment:
Wherever the double cleaner runs, I think you should add a comment explaining why

Reply:
added